include/core/thermal_expansion_model.h

```diff
@@ -0,0 +1,113 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#ifndef lethe_thermal_expansion_model_h
+#define lethe_thermal_expansion_model_h
+
+#include <core/physical_property_model.h>
+
+/**
+ * @brief ThermalExpansionModel. Abstract class that allows to calculate the
```

Comment:
So it's abstract, just as `AuxiliaryPhysics`? If yes, should not the methods be `virtual`?

Reply:
I don't think the virtual keyword here is necessary. Since the class does not implement the value, vector_value and etc. of the base class, I don't think it could be instantiated (although I might be wrong ) :).